presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctionsBase.java

```diff
@@ -193,6 +194,14 @@ public void testFromUnixTimeWithOffset()
         assertFunction("from_unixtime(" + seconds + ", " + timeZoneHoursOffset + ", " + timezoneMinutesOffset + ")", TIMESTAMP_WITH_TIME_ZONE, toTimestampWithTimeZone(expected));
     }
 
+    @Test
+    public void testFromUnixTimeWithOffsetNegative()
+    {
+        assertInvalidFunction("from_unixtime(0, 1, 10000)", INVALID_FUNCTION_ARGUMENT);
```

Comment:
Please move these three tests to the end of the `testFromUnixTimeWithOffset` method above, and add a comment before these that says `// test invalid minute offsets`.